prow/plugins/config.go

```diff
@@ -427,10 +428,21 @@ type ConfigMapSpec struct {
 	// Clusters is a map from cluster to namespaces
 	// which specifies the targets the configMap needs to be deployed, i.e., each namespace in map[cluster]
 	Clusters map[string][]string `json:"clusters"`
```

Comment:
Apparently we now have
* `Namespace`
* `Namespaces`
* `AdditionalNamespaces`
* `Clusters`
* `ClusterGroups`

This seems a little redundant. Thoughts on combining namespace and cluster into context?

Reply:
Yeah, this is super convoluted and would make sense to clean up, however I am a bit concerned that that might break ppl. 

> Thoughts on combining namespace and cluster into context?

`namespaces` is not actually user-settable and `clusters` is a `map[clustername][]clusterNamespace` so pretty much what you meant by `context` already?

Reply:
Could we just drop namespace and additional namespaces? Either now or in the future? Thoughts on naming clusters context since it is essentially a context already?

Reply:
> Could we just drop namespace and additional namespaces? 

Funnily they have been deprecated already in october 2019, but without removal timeline. I have updated that to say that they will be removed in october 2020.

> Thoughts on naming clusters context since it is essentially a context already?
Its not completely the same, because a cluster allows to specify multiple namespaces, which a context would normally not allow. Given that and that the renaming is a hassle with little benefits, I would keep that as-is.